telemetry/ratelimiter.go

```diff
@@ -99,7 +99,8 @@ func (rl *RateLimiter) cleanup(ctx context.Context, cleanupEvery time.Duration)
 			return
 		case now := <-t.C:
 			numCleaned := 0
-			for ip, limiter := range rl.values() {
+			limiters := rl.values()
```

Comment:
@richard-ramos unrelated to the refactoring, but the CI failed, so I added the fix

https://github.com/status-im/telemetry/actions/runs/10353387310/job/28656165988?pr=37